frontend/app/src/pages/settings/api-keys/external/index.vue

```diff
@@ -130,33 +153,43 @@ onMounted(async () => {
     </template>
 
     <api-key-box>
-      <service-key
-        v-model="etherscanKey"
-        class="external-services__etherscan-key"
-        :title="tc('external_services.etherscan.title')"
-        :description="tc('external_services.etherscan.description')"
-        :label="tc('external_services.etherscan.label')"
-        :hint="tc('external_services.etherscan.hint')"
-        :loading="loading"
-        :tooltip="tc('external_services.etherscan.delete_tooltip')"
-        @save="save('etherscan', $event)"
-        @delete-key="showConfirmation('etherscan')"
-      />
-    </api-key-box>
-
-    <api-key-box>
-      <service-key
-        v-model="optimismEtherscanKey"
-        class="external-services__optimism-etherscan-key"
-        :title="tc('external_services.optimism_etherscan.title')"
-        :description="tc('external_services.optimism_etherscan.description')"
-        :label="tc('external_services.optimism_etherscan.label')"
-        :hint="tc('external_services.optimism_etherscan.hint')"
-        :loading="loading"
-        :tooltip="tc('external_services.optimism_etherscan.delete_tooltip')"
-        @save="save('optimism_etherscan', $event)"
-        @delete-key="showConfirmation('optimism_etherscan')"
-      />
+      <v-card flat>
+        <v-card-title>
+          {{ tc('external_services.etherscan.title') }}
+        </v-card-title>
+        <v-card-subtitle>
+          {{ tc('external_services.etherscan.description') }}
+        </v-card-subtitle>
+      </v-card>
+      <v-tabs v-model="evmEtherscanTabIndex">
+        <v-tab v-for="tab in evmEtherscanTabs" :key="tab.chain">
+          <evm-chain-icon :chain="tab.chain" />
+          <div class="ml-2">{{ tab.chain }}</div>
+        </v-tab>
+      </v-tabs>
+      <v-divider />
+      <v-tabs-items v-model="evmEtherscanTabIndex" class="pt-4">
+        <v-tab-item v-for="tab in evmEtherscanTabs" :key="tab.chain">
+          <service-key
+            v-model="tab.ref.value"
```

Comment:
Can we find an alternative approach here?
In general, if we have to manually unref a ref in a template then we are doing something wrong.